pkg/db/db.go

```diff
@@ -53,7 +53,9 @@ type Operation interface {
 }
 
 type Getter interface {
-	Get(string, string) ([]types.Advisory, error)
+	// Get uses the following parameters:
+	// `osVer`, `pkgName`, `arch`
+	Get(string, string, string) ([]types.Advisory, error)
```

Comment:
We may add more params in the future. What if using a struct?

```suggestion
	Get(GetParams) ([]types.Advisory, error)
```

Reply:
This makes sense - in this case we won't have to update the Trivy code (after adding new parameters).
Updated in https://github.com/aquasecurity/trivy-db/pull/558/commits/f95b696a092b777963efc6a78d0f7e29bacf2324

Reply:
This makes sense - in this case we won't have to update the Trivy code (after adding new parameters).
Updated in https://github.com/aquasecurity/trivy-db/pull/558/commits/c78fde1c884db57f7c42b9f5e025c34a0cd22c2b